build.gradle.kts

```diff
@@ -22,12 +22,9 @@ apply<GradleVersionsFilterPlugin>()
 apply<VersionCatalogUpdatePlugin>()
 apply<TaskTreePlugin>()
 
-if (JavaVersion.current() != JavaVersion.VERSION_11 && JavaVersion.current() != JavaVersion.VERSION_17) {
-  throw GradleException(
-    """Project must be built with JDK 11 or 17 since:
-    1. as of v3.24, Checker Framework only supports JDK 8, 11 and 17: https://checkerframework.org/manual/#installation
-    2. codebase is Java 11-compatible, so can't be built on JDK 8"""
-  )
+// TODO (#983): use Gradle toolchain instead
```

Comment:
Done on #1202

Reply:
Yes, so - do we need this change in `build.gradle.kts` in this pull request? This check will be removed anyway after merging #1202 ?